### DISCUSSION TITLE
Power Automate Kick-off!

### DISCUSSION THREAD

**Discussion by fboucher** [microsoft/Power-Platform-For-Beginners#5] [Power Automate]
Do you have a section of the [template](https://github.com/microsoft/Power-Platform-For-Beginners/tree/main/templates) you would like to start working on?
Let your teammates know!

Then you can **Fork** the repo and start creating content!

**Comment by MarceLehmann** [comment]
1. Introduction to process automation and workflow concepts.
2. Creating automated flows using pre-built connectors and triggers.
3. Implementing business logic with conditions, loops, and expressions.
4. Building approval processes and notifications.
5. Developing custom connectors and actions.

@Pieter: I would like to do tasks 2 and 3, if that's okay with you? If you'd prefer me to take a third task, then I would be happy to take either task 1 or 4.
@FBoucher: Are these 5 themes correct? What's the timeline?

**Reply by fboucher** [reply]
I like it @MarceLehmann! 🔥  What would be that project? 
Keep in mind that we are trying to be project-based lesson? Imagine building something in a step-by-step approach. Having a defined project/ goal will help the to get consistency between sections.

**Reply by MarceLehmann** [reply]
@FBoucher
Would it be best if everyone had the same project, or would it be easier and better if everyone had different projects? In other words, should the projects be interconnected, or should they be completely separate from each other?

**Reply by fboucher** [reply]
@MarceLehmann I think it will be easier to have lesson specific projects. By that I mean 1 project for the Power Automate lesson.

the other teams working on the other lessons will create their project. This way you don't have any "dependence" on another lessons. IF I just want to do the Power Automate one, I don't need to do something else first.

**Reply by MarceLehmann** [reply]
@FBoucher
And should points 1 through 5 from above all be part of the same project?

**Reply by fboucher** [reply]
YES
All those are for the same lesson (module, section) that focuses on Power Automate.

**Reply by PowerRobin** [reply]
I see that dependencies would complicate things, but having one big project through the whole thing would be a better experience in my opinion

**Reply by fboucher** [reply]
I agree @PowerRobin it could be interesting. However, I want to keep the creation process simple. Once it's all completed, we will see what we can do to make them part of a same story.

**Reply by Pieter-Veenstra** [reply]
@MarceLehmann I'm happy to pick up any of the items. So if you are happy to pick up 2 and 3 then I'm happy to start with 1, 4 and/or 5. 

It would probably help though to have a look at how deep we want to go with all of the topics. Also, do we have any guidelines on standards that we want to follow.

**Comment by fboucher** [comment]
I'll close this discussion now that most happens on Teams and keep only the more general ones open for now. :)
